src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -1,35 +1,15 @@
 package seedu.address.logic.commands;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.commons.core.Messages.MESSAGE_CUSTOMERS_LISTED_OVERVIEW;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.testutil.TypicalCustomers.CARL;
-import static seedu.address.testutil.TypicalCustomers.ELLE;
-import static seedu.address.testutil.TypicalCustomers.FIONA;
-import static seedu.address.testutil.TypicalCustomers.getTypicalAddressBook;
-
-import java.util.Arrays;
-import java.util.Collections;
-
-import org.junit.jupiter.api.Test;
-
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.UserPrefs;
-import seedu.address.model.customer.NameContainsKeywordsPredicate;
-
 /**
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
```

Comment:
Small comment, please add a TODO here for me to write new test cases here pls;
Example `TODO: Write new test cases for the overhauled command`

Reply:
Acceptable